### ISSUE TITLE
add Gradio Web Demo to cvpr 2022 organization

### ISSUE THREAD

**Issue by AK391** [wpeebles/gangealing#24]
Hi, would you be interested in adding gangealing to Hugging Face as a Gradio Web Demo for CVPR 2022? The Hub offers free hosting, and it would make your work more accessible and visible to the rest of the ML community. Models/datasets/spaces(web demos) can be added to a user account or organization similar to github.

more info see CVPR organization on Hugging Face: https://huggingface.co/CVPR

here is a example Gradio Demo for the CVPR org: https://huggingface.co/spaces/CVPR/ml-talking-face

and here is a guide for adding web demo to the organization: https://huggingface.co/blog/gradio-spaces

Please let us know if you would be interested and if you have any questions, we can also help with the technical implementation.

**Comment by wpeebles** [comment]
Hi @AK391, thanks for the suggestion! It would be cool to host a demo on Huggingface. Do you know if it's possible to deploy demos to Spaces with a GPU backend? We use a custom splatting op in our colab demo that currently requires CUDA, although I guess it should be possible to write a CPU-based extension too.

**Comment by AK391** [comment]
@wpeebles yes a gpu can be assigned to a spaces based on availability, you can open a discussion like this https://huggingface.co/spaces/CVPR/LIVE/discussions/1 similar to github and tag @osanseviero, thanks!
